blog/_posts/2018-07-09-introducing-TeamBarcelonaDuo.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Introducing Team Barcelona Duo
+layout: post
+created_at: Mon July 09 2018
+permalink: blog/introducing-TeamBarcelonaDuo
+current: blog
+author: Lori and Amalia
+twitter: HarlemExpat
+categories:
+- student-posts
+---
+Hello everyone, we are Lori and Amalia, two amazing ex-sociologist turned programmers. We are Team BarcelonaDuo. We met at [codebar](https://codebar.io/) <br> and are currently organizers there.
+
+Codebar is a non-profit organization that runs workshops for underrepresented groups in tech. The aim is to increase interest and diversity in programming.
+
+![Lori and Amalia!](/img/blog/2018/BarcelonaDuo1.png)
+<div class="image-credits"><b>Lori and Amalia at Thoughtworks (image by Amalia Cardenas)</b></div>
+
+
+### Lori
+I've been working as an EFL teacher in Spain for several years now. In fact, my first experience
+with coding came from looking for a way to motivate my students. I figured I could teach them
+Scratch and from there they could make interactive stories or games. What I didn't know at the time
+was that programming involves research, problem solving, and learning -- a type of mental stimulation
+that I hadn't felt in years -- and I was hooked.
+
+![Lori studying code!](/img/blog/2018/BarcelonaDuo5.jpg)
+<div class="image-credits"><b>Lori studying code  (image by Amalia Cardenas)</b></div>
+<br>
+I got a bit more serious about coding a few years latter. I moved to Barcelona, in part for an opportunity to meet other programming hobbyist. I started attending codebar and met
```

Comment:
For the consistency purpose in the file, can you please add the remaining text ('...an amazing group...') in the same line as its a same sentence?